### PR TITLE
[RFC] man.vim: remove newline from man errors

### PR DESCRIPTION
**Pull request by nhooyr**
Otherwise errors appear like:

screen shot 2017-01-01 at 1 29 48 am

### CODE REVIEW

File: runtime/autoload/man.vim [hunk 1]
```diff
@@ -106,7 +106,7 @@ function! s:system(cmd, ...) abort
     throw printf('command interrupted: %s', join(a:cmd))
   endif
   if opts.exit_code != 0
-    throw printf("command error (%d) %s: %s", jobid, join(a:cmd), opts.stderr)
+    throw printf("command error (%d) %s: %s", jobid, join(a:cmd), opts.stderr[:len(opts.stderr)-2])
```
Comment (justinmk) [inline runtime/autoload/man.vim]: Would be better to remove  whitespace:

    substitute(opts.stderr, '\s\+$', 'g')

Comment (nhooyr) [inline runtime/autoload/man.vim]: Wouldn't that also remove all the spaces?

Comment (nhooyr) [inline runtime/autoload/man.vim]: oh, nevermind, I see now. trailing whitespace.

**Comment by nhooyr** [comment]
Fixed.

**Pull request merged**